web/src/main/react-dashboard/src/components/TestRunView.js

```diff
@@ -49,563 +42,570 @@ import {FAIL, SUCCESS, ERROR, PENDING, RUNNING, HTTP_UNAUTHORIZED, LOGIN_URI, TE
  */
 class TestRunView extends Component {
 
-  constructor(props) {
-    super(props);
-    this.state = {
-      testScenarioSummaries: [],
-      scenarioTestCaseEntries: [],
-      testSummaryLoadStatus: PENDING,
-      logContent: "",
-      logDownloadStatus: PENDING,
-      isLogTruncated: false,
-      inputStreamSize: "",
-      showLogDownloadErrorDialog: false
-    };
-  }
+    constructor(props) {
+        super(props);
+        this.state = {
+            testScenarioSummaries: [],
+            scenarioTestCaseEntries: [],
+            testSummaryLoadStatus: PENDING,
```

Comment:
PENDING status means that the test is still running. Does it make sense to change the default status to DID_NOT_RUN which shows a gray-colored icon? When the actual data comes, we can populate with correct properties. wdyt?

Reply:
+1 It is a better approach and lets raise a separate ticket and handle this as well.